css/style.css

```diff
@@ -179,6 +179,238 @@
   .arrow-downd:hover {
     transform: scale(1.2);
   }
+
+  .future-drive {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .future-drive__title {
+    display: none;
+    position: absolute;
+    margin: 0 50px 0 auto;
+    left: 420px;
+    font-size: 70px;
+    line-height: 85px;
+    color: #e51937;
+  }
+
+  .future-drive__img {
+    width: 100%;
+  }
+
+  .future {
+    background-color: #e51937;
+    color: #fff;
+  }
+
+  .container-future {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    padding-top: 20px;
+  }
+
+  .tittle-article {
+    margin: 0 0 5px 0;
+    font-size: 30px;
+    line-height: 37px;
+  }
+
+  .article-text {
+    max-width: 300px;
+    font-size: 15px;
+    line-height: 18px;
+    font-weight: lighter;
+  }
+
+  .future-drive-list {
+    display: none;
+  }
+
+  .future-drive-list__item {
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+  }
+
+  .fruture-text {
+    margin: 0;
+    font-size: 35px;
+    line-height: 43px;
+    font-weight: bold;
+  }
+
+  .fruture-text-description {
+    margin: 0;
+    font-size: 20px;
+    line-height: 24px;
+    font-weight: normal;
+  }
+
+  .fruture-description {
+    max-width: 140px;
+    font-size: 12px;
+    line-height: 15px;
+    font-weight: lighter;
+  }
+
+  .fruture-text-description-wifi {
+    margin: 0;
+    max-width: 150px;
+    font-size: 20px;
+    line-height: 24px;
+    font-weight: normal;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/GurevAleksey_itogovaya/pullRequest?prid=6120148)